core/src/main/java/com/graphhopper/routing/lm/PrepareLandmarks.java

```diff
@@ -122,12 +122,10 @@ public boolean loadExisting() {
     }
 
     @Override
-    public void doWork() {
-        super.doWork();
-
+    public void doSpecificWork() {
         StopWatch sw = new StopWatch().start();
-        LOGGER.info("Start calculating " + lms.getLandmarkCount() + " landmarks, default active lms:"
-                + defaultActiveLandmarks + ", weighting:" + lms.getLmSelectionWeighting() + ", " + Helper.getMemInfo());
+        LOGGER.info("Start calculating {} landmarks, default active lms: {}, weighting: {}, {}",
+                lms.getLandmarkCount(), defaultActiveLandmarks, lms.getLmSelectionWeighting(), Helper.getMemInfo());
```

Comment:
To be honest: I found the previous logging statement more clear :)

The "{}" notation is used to avoid string concatening if we disable logging for this class, but usually the advantage is only for massive logging like we could get in the REST server but even there we would not deactivate logging because it is valuable to have them for analyzation

Reply:
I was not trying to improve performance here, but think {} is more readable, but that's probably because I am used to it. I will revert it, don't have a strong opinion here.

Reply:
The string itself is more clear. But counting the brackets {} {} {} to know which property goes where makes it harder for me. Also not a strong just a minor opinion :)